src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -58,7 +58,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     if (user == null || user.equals("")) {
       // Request is invalid, return empty array
-      response.getWriter().println("[]");
+      response.getWriter().println("This is null... Are you logged in?");
```

Comment:
Does this cause any problems when the JavaScript tries to parse this as an array?

Reply:
Not sure. But I'll change it back just in case.